subprojects/core/src/integTest/groovy/org/gradle/api/ExternalScriptExecutionIntegrationSpec.groovy

```diff
@@ -17,15 +17,20 @@ package org.gradle.api
 
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.test.fixtures.server.http.HttpServer
+import spock.lang.Unroll
 
 class ExternalScriptExecutionIntegrationSpec extends AbstractIntegrationSpec {
```

Comment:
❌ This is missing coverage for `https` served resources. We probably don't need every test to run with both HTTP and HTTPS.

With HTTPS, we do need tests for failing/wrong credentials.